wls-eai-service/src/main/resources/db/dummydata/h2/V1_1__wahlvorstandTestdaten.sql

```diff
@@ -1,74 +1,74 @@
-INSERT INTO wahlvorstand
+INSERT INTO wahlvorstand (id, wahlbezirkID)
 VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0000-0000-000000000001');
-INSERT INTO wahlvorstandsmitglied
+INSERT INTO wahlvorstandsmitglied (id, vorname, nachname, funktion, anwesend, wahlvorstandID, anwesenheitUpdatedOn) (id, vorname, nachname, funktion, anwesend, wahlvorstandID, anwesenheitUpdatedOn)
```

Comment:
Ist das nicht einmal zu viel mit den Spaltennamen in den Klammern?

Reply:
upsi - fixed mit ae46b8ed376e34aa05fefa43fe880e7469d7d6c4

mit 5ee8c413c68a0b3bce5644fe3d1b86c6883e52f0 habe ich auch einen Test erstellt der zumindest für h2 testet dass die Dummydaten korrekt sind.